cli/aws_orbit/models/context.py

```diff
@@ -561,6 +565,57 @@ def fetch_toolkit_data(context: V) -> None:
         top_level = "orbit" if isinstance(context, Context) else "orbit-f"
 
         resp_type = Dict[str, List[Dict[str, List[Dict[str, str]]]]]
+
+        slrt_toolkit_name: str = "orbit"
+        slrt_toolkit_stack_name: str = f"softwarelabs-remote-toolkit-{slrt_toolkit_name}"
```

Comment:
the cfn module included with the toolkit has a get_stack_name() function that will return the stack name, rather than trying to generate it